### ISSUE TITLE
There is typo annotation in a code

### ISSUE THREAD

**Issue by alu** [doctrine/common#776]
There is `@Param` annotation here.

https://github.com/doctrine/common/blob/2.7/lib/Doctrine/Common/Proxy/ProxyGenerator.php#L1016

I think the annotation typo of `@param`.

I notice when catched an exception such as below.

```
[Semantical Error] The annotation "@Param" in method Doctrine\Common\Proxy\ProxyGenerator::getMethodReturnType() was never imported. Did you maybe forget to add a "use" statement for this annotation?" at /n/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php line 54
```

Thanks.

**Comment by dao** [comment]
Shouldn't this fix get a 2.7.3 tag? 2.7.2 is just broken as is

**Comment by Ocramius** [comment]
@dao no, it's not broken - something is simply scanning for annotations in the wrong directory there, if you are experiencing this crash.

This change will simply land in 2.8.0.
